frontend/src/shared/GameButton.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import {useNavigate} from "react-router-dom";
+
+export default function GameButton() {
+const navigate=useNavigate();
+    return (
+        <div className={"GameButton"}>
+            <button id={"startButton"}  onClick={() => {navigate("/zooGame")
```

Comment:
Hier gerne den Vorschlag von Irene berücksichtigen.
Einen "PLAY" Button, während man bereits spielt zu haben, macht in meinen Augen auch weniger Sinn :-)

Reply:
ist gefixt